docs/src/manual/boundary_conditions.md

```diff
@@ -260,3 +260,29 @@ pdbc = PeriodicDirichlet(
         (x, t) -> ū + ∇ū  ⋅ (x - x̄)
     )
     ```
+
+# Initial Conditions
+
+When solving time-dependent problems, initial conditions, different from zero, may be required. 
+For finite element formulations of ODE-type, 
+i.e. ``\boldsymbol{x}'(t) = \boldsymbol{f}(\boldsymbol{x}(t),t)``, 
+where ``\boldsymbol{x}(t)`` are the degrees of freedom,
+initial conditions can be specified by the [`apply_analytical!`](@ref) function.
+For example, specify the initial pressure as a function of the y-coordinate
+```julia
+ρ = 1000; g=9.81    # density [kg/m³] and gravity [N/kg]
+grid = generate_grid(Quadrilateral, (10,10))
+dh = DofHandler(grid); push!(dh, :u, 2); push!(dh, :p, 1); close!(dh)
+x = zeros(ndofs(dh))
+apply_analytical!(x, dh, :p, x->ρ*g*x[2])
+```
+
+See also [Time Dependent Problems](@ref) for one example. 
+
+*Note about solving DAE:* 
+A Differential Algebraic Equations (DAE) is an equation of the form
+``\boldsymbol{r}(\boldsymbol{x}(t),\boldsymbol{x}'(t),t)=\boldsymbol{0}``,
+which cannot be expressed as an ODE.
```

Comment:
Maybe we can be a bit more precise here and give a quick example. My idea is as follows.

```suggestion
A Differential Algebraic Equations (DAE) is an equation of the form
``\boldsymbol{r}(\boldsymbol{x}(t),\boldsymbol{x}'(t),t)=\boldsymbol{0}``,
which usually cannot be expressed as a true ODE. They occur often,
but not always, in forms where some time derivatives are missing
``
x_1'(t) = f(\boldsymbol{x}(t),t)
0 = g(\boldsymbol{x}(t),t)`
``
```